lib/util/zip.ex

```diff
@@ -113,6 +118,24 @@ defmodule Antikythera.Zip do
     end
   end
 
+  defunp validate_path_exists(path :: v[String.t]) :: :ok | {:error, tuple} do
+    if File.exists?(path) do
+      :ok
+    else
+      {:error, {:not_found, %{path: path}}}
+    end
+  end
+
+  defunp validate_suffix(src_raw_path :: v[String.t], cwd_path :: v[String.t]) :: :ok | {:error, tuple} do
+    suffixed = String.ends_with?(src_raw_path, "/")
+    path = Path.expand(src_raw_path, cwd_path)
```

Comment:
There are two formats of relationship between names,
- raw_path -> path
- path -> full_path

which are the same in meaning.
I will unify expression to the latter.

Reply:
Fixed naming rule in c78679f.